### PR TITLE
Remove featureSRSConsistency test

### LINKED ISSUE #78: No requirement can be found for test "feature S R Sconsistency"
Test: https://github.com/opengeospatial/ets-gpkg12/blob/477026324c88313316275fd19d74014be164641d/src/main/java/org/opengis/cite/gpkg12/features/FeaturesTests.java#L304-L361

Test fails because the results of two sql statements are not equal (first if):
 * SELECT count(*) as ggcCount FROM 'gpkg_geometry_columns';
   * returns 68
 * SELECT count(*) as ggcCount FROM 'gpkg_contents' WHERE data_type = 'features';
   * returns 34

The test expects that both counts are equal.

In http://www.geopackage.org/spec/ I cannot find a test case with id `/base/core/contents/data/table_def/srs_id`, and I don't see that Requirement 13 makes the test necessary. 
Furthermore the if-else-statement in the end of the test can be simplified.

### PR DESCRIPTION
**Pull request by ajanett**
as it is now covered by test
featureGeometryColumnsDataValuesSrsID (Requirement 146)

Update POM to reference TEAM Engine 5.3 as an override to ets-common v9

These changes allow the closure of issue: https://github.com/opengeospatial/ets-gpkg12/issues/78

Fixes #78

### CODE REVIEW

**Comment by lgoltz** [comment]
@ajanett Thanks for your investigation!
Can you remove the change in the pom.xml?

File: pom.xml [hunk 1]
```diff
@@ -57,6 +57,7 @@
     <dependency>
       <groupId>org.opengis.cite.teamengine</groupId>
       <artifactId>teamengine-spi</artifactId>
+      <version>${docker.teamengine.version}</version>
```
Comment (lgoltz) [inline pom.xml]: @ajanett Can you revert this change? The docker.teamengine.version should be used only by the docker plugin.

**Comment by dstenger** [comment]
Duplicate of https://github.com/opengeospatial/ets-gpkg12/pull/110.
